staffs/serializer.py

```
@@ -1,11 +1,83 @@
+
     class Meta(BaseSerializer.Meta):
         model = models.Teacher
         fields = '__all__'
 
+        validators = [
+            serializers.UniqueTogetherValidator(
+                queryset=models.Teacher.objects.exclude(status='i'),
+                fields=['user'],
+                message="An active teacher profile already exists for this user."
+            )
+        ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)